docs/content/docs/test_framework/Mocking.md

```diff
@@ -121,6 +121,25 @@ This annotation is available in all types of tests but make sense mostly for fun
 class MyTest { ... }
 ```
 
+## Mocking Preferences
+
+All kind of tests are using real implementations of `com.avito.android.util.preferences.Preferences`, not mocked ones.
+
+All entities that depend on `Preferences` are gathered together in `PreferencesRule`. You can access that entities without going into details of `Preferences`
+implementations/mocks like the following:
+
+```kotlin
+@get:Rule
+val preferencesRule = PreferencesRule()
+
+@Test
+fun profileTest() {
+    // ...
+    preferencesRule.mutableProfileInfoStorage.saveProfileInfo(info)
```

Comment:
What is a `mutableProfileInfoStorage`? Who does write it? How do i need to use it?